tox.cover.sh

```diff
@@ -31,6 +31,8 @@ cover () {
     min=98
   elif [ "$1" = "certbot_dns_dnsmadeeasy" ]; then
     min=99
+  elif [ "$1" = "certbot_dns_gehirn" ]; then
```

Comment:
You'll also want to add this package to line 12, so that it's included in the coverage check by default.